pyppeteer/page.py

```diff
@@ -201,21 +184,89 @@ def _onTargetDetached(event: Dict) -> None:
         client.on('Runtime.exceptionThrown',
                   lambda exception: self._handleException(
                       exception.get('exceptionDetails')))
-        client.on('Security.certificateError',
-                  lambda event: self._onCertificateError(event))
         client.on('Inspector.targetCrashed',
                   lambda event: self._onTargetCrashed())
         client.on('Performance.metrics',
                   lambda event: self._emitMetrics(event))
         client.on('Log.entryAdded',
                   lambda event: self._onLogEntryAdded(event))
+        client.on('Page.fileChooserOpened',
+                  lambda event: self._onFileChooser(event))
 
         def closed(fut: asyncio.futures.Future) -> None:
```

Comment:
If this arg is unused, I'd prefer that we use the `_` convention for an unused variable (in which case we don't even have to type it, but it doesn't hurt to anyway)
```suggestion
        def closed(_)-> None:
```